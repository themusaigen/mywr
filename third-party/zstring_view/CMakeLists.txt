include(FetchContent)

FetchContent_Declare(
  "zstring_view"
  GIT_REPOSITORY "https://github.com/moonpotato/zstring_view.git")

FetchContent_MakeAvailable(zstring_view)

add_library(zstring_view INTERFACE)

target_include_directories(zstring_view INTERFACE "${zstring_view_SOURCE_DIR}")
