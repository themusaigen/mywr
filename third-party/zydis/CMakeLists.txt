include(FetchContent)

FetchContent_Declare(
  "zydis"
  GIT_REPOSITORY "https://github.com/zyantific/zydis.git"
  GIT_TAG "v4.1.1")

FetchContent_MakeAvailable(zydis)
