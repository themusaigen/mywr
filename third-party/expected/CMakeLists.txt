include("FetchContent")

FetchContent_Declare(
  "expected"
  GIT_REPOSITORY "https://github.com/TartanLlama/expected.git"
  GIT_TAG "v1.1.0")

FetchContent_MakeAvailable("expected")
